### PR TITLE
plotlyJS

### PR DESCRIPTION
**Pull request by sglyon**
The goal of this PR is to replace some of the more verbose "by hand" figures being used in `dcc_graph` with ones produced by PlotlyJS.jl

Most of the change are in the user guide/tutorial section at the beginning of the docs. However, there are a couple examples in the dash_core_components section also.

Besides making those changes -- I cleaned up a bit of the language and found a handful of bugs in the code examples.

I also delete some Python and R files that I didn't believe belonged there -- but if I was mistaken on that please let me know and we will restore them!

### CODE REVIEW

**Review by efmanu — APPROVED** [review]

File: DashDocs/src/chapters/basic_callbacks/examples/hello-slider.jl [hunk 1]
```diff
@@ -24,24 +23,21 @@ callback!(
...
     Input("year-slider-1", "value"),
 ) do selected_year
-    return Plot(
+    return plot(
         df1[df1.year .== selected_year, :],
         Layout(
             xaxis_type = "log",
-            xaxis_title = "GDP Per Capita",
-            yaxis_title = "Life Expectancy",
+            xaxis_title_text = "GDP Per Capita",
```
Comment (mohdibntarek) [inline DashDocs/src/chapters/basic_callbacks/examples/hello-slider.jl]: Was the old one wrong?

File: DashDocs/src/chapters/basic_callbacks/examples/multi-inputs.jl [hunk 2]
```diff
@@ -68,26 +70,20 @@ callback!(
...
     df2f = df2[df2.year .== year_value, :]
-    return Plot(
-        df2f[df2f[!, Symbol("Indicator Name")] .== xaxis_column_name, :Value],
-        df2f[df2f[!, Symbol("Indicator Name")] .== yaxis_column_name, :Value],
+    x = df2f[df2f[!, Symbol("Indicator Name")] .== xaxis_column_name, :Value]
+    y = df2f[df2f[!, Symbol("Indicator Name")] .== yaxis_column_name, :Value]
+    return plot(
+        x, y,
         Layout(
-            xaxis_type = xaxis_type == "Linear" ? "linear" : "log",
```
Comment (mohdibntarek) [inline DashDocs/src/chapters/basic_callbacks/examples/multi-inputs.jl:75]: Was this not functional?

Comment (mohdibntarek) [reply DashDocs/src/chapters/basic_callbacks/examples/multi-inputs.jl:75]: Ah I see, nvm that comment.

File: DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl [hunk 3]
```diff
@@ -1,22 +0,0 @@
-using Dash, DashCoreComponents
```
Comment (mohdibntarek) [inline DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl:1]: Why remove this file?

**Comment by sglyon** [comment]
ref https://github.com/plotly/Dash.jl/issues/88

File: DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl (hunk 3 above)
Comment (sglyon) [reply DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl:1]: Did not seem to be included anywhere. I can re-add if I was wrong

**Review by mohdibntarek — APPROVED** [review]

File: DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl (hunk 3 above)
Comment (mohdibntarek) [reply DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl:1]: Yes we need this file. It seems to be included too https://github.com/plotly/dash-julia-docs/pull/20/files#diff-0d967af92fd3f1cafcbd1d027fd2340b459bb625a5ec387337b1fa864713aba6R4.

Comment (mohdibntarek) [reply DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl:1]: Please add it back then I am happy to merge this. Everything else looks good.

File: Manifest.toml [hunk 4]
```diff
@@ -33,50 +45,62 @@ git-tree-sha1 = "ec77888ac3e78f9d372c2b533bdb52668f9e2b09"
...
 [[Crayons]]
 git-tree-sha1 = "3f71217b538d7aaee0b69ab47d9b7724ca8afa0d"
 uuid = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
 version = "4.0.4"
 
 [[Dash]]
-deps = ["CodecZlib", "DashBase", "DashCoreComponents", "DashHtmlComponents", "DashTable", "DataStructures", "HTTP", "JSON", "JSON2", "MD5", "PlotlyBase", "Sockets", "Test", "UUIDs"]
-git-tree-sha1 = "7c09315c7dbfd9510dd822fcee3910c4c139afd8"
+deps = ["CodecZlib", "DashBase", "DashCoreComponents", "DashHtmlComponents", "DashTable", "DataStructures", "HTTP", "JSON", "JSON2", "MD5", "PlotlyBase", "Requires", "Sockets", "Test", "UUIDs"]
+path = "/home/sglyon/.julia/dev/Dash"
```
Comment (efmanu) [inline Manifest.toml]: Can you the actual Dash package in Julia registry instead of dev package?

File: Manifest.toml [hunk 5]
```diff
@@ -98,9 +122,9 @@ version = "0.1.0"
 
 [[DashCoreComponents]]
 deps = ["DashBase"]
-git-tree-sha1 = "4f4935719119350b1f4c464341ef64743920ea74"
+path = "/home/sglyon/.julia/dev/DashCoreComponents"
```
Comment (efmanu) [inline Manifest.toml]: Can you the actual DashCoreComponents package in Julia registry instead of dev package?

**Review by efmanu — CHANGES_REQUESTED** [review]

File: Manifest.toml (hunk 5 above)
Comment (sglyon) [reply Manifest.toml]: Good catch -- will fix! Waiting on Dash.jl 0.1.6 to be released and will then make the change and push here

File: Manifest.toml (hunk 4 above)
Comment (sglyon) [reply Manifest.toml]: Good catch -- will fix! Waiting on Dash.jl 0.1.6 to be released and will then make the change and push here

File: DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl (hunk 3 above)
Comment (sglyon) [reply DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl:1]: Thanks -- added back locally. Waiting on v0.1.6 to be released and will rebase this PR off that and push change

File: Manifest.toml (hunk 4 above)
Comment (sglyon) [reply Manifest.toml]: fixed!

File: Manifest.toml (hunk 5 above)
Comment (sglyon) [reply Manifest.toml]: fixed!

**Comment by sglyon** [comment]
ok I think this is ready to go!

**Pull request merged**